src/frontend/templatehelpers/templatetags/custom_url_tags.py

```diff
@@ -0,0 +1,28 @@
+from django import template
+
+register = template.Library()
+
+# These are performance hacks that directly build a relative URL for certain pages
+# instead of relying on django reverse(), which seems to be very slow when called
+# hundreds of times, once for each different item in a lst.
+
+@register.simple_tag
+def training_games_list_by_network(run_name, network_name):
```

Comment:
Nice to know.

Do you know the performance impact of just this ?

Reply:
This one is like 50 or 70 milliseconds or something like that, when there is a lot of items in the table. Apparently reverse URL lookup is very slow.